src/public/V8JsiRuntime.h

```diff
@@ -54,6 +54,9 @@ struct V8RuntimeArgs {
   size_t initial_heap_size_in_bytes{0};
   size_t maximum_heap_size_in_bytes{0};
 
+  // Set this to override the target name displayed in the debugger (to distinguish multiple parallel runtimes)
+  std::string debuggerRuntimeName;

```

Comment:
>debuggerRuntimeName [](http://example.com/codeflow?start=14&length=19)

(nit) where is this being set?

Reply:
Or there will be a separate CR to set it in RN?

Reply:
Yes, this needs to come in from external code using the V8Runtime (like RNW or devmain)